src/core/function_admin.tf

```diff
@@ -0,0 +1,407 @@
+  runtime_version  = "~4"
+
+  always_on                                = "true"
+  application_insights_instrumentation_key = data.azurerm_application_insights.application_insights.instrumentation_key
+
+  app_service_plan_info = {
+    kind                         = var.function_admin_kind
+    sku_tier                     = var.function_admin_sku_tier
+    sku_size                     = var.function_admin_sku_size
+    maximum_elastic_worker_count = 0
+  }
+
+  app_settings = merge(
+    local.function_admin.app_settings_common, {
+      # TODO: All of this can be removed?
+      "AzureWebJobs.UpdateVisibleServicesCache.Disabled"             = "1"
+      "AzureWebJobs.UpdateVisibleServicesCacheOrchestrator.Disabled" = "1"
+      "AzureWebJobs.UpdateVisibleServicesCacheActivity.Disabled"     = "1"
+    }
+  )
+
+  internal_storage = {
+    "enable"                     = true,
+    "private_endpoint_subnet_id" = data.azurerm_subnet.private_endpoints_subnet.id,
+    "private_dns_zone_blob_ids"  = [data.azurerm_private_dns_zone.privatelink_blob_core_windows_net.id],
+    "private_dns_zone_queue_ids" = [data.azurerm_private_dns_zone.privatelink_queue_core_windows_net.id],
+    "private_dns_zone_table_ids" = [data.azurerm_private_dns_zone.privatelink_table_core_windows_net.id],
+    "queues"                     = [],
+    "containers"                 = [],
+    "blobs_retention_days"       = 0,
+  }
+
+  subnet_id = module.admin_snet.id
+
+  allowed_subnets = [
+    module.admin_snet.id,
+    module.apim_snet.id,
+  ]
+
+  tags = var.tags
+}
+
+module "function_admin_staging_slot" {
+  source = "git::https://github.com/pagopa/azurerm.git//function_app_slot?ref=v3.4.0"
+
+  name                = "staging"
+  location            = var.location
+  resource_group_name = azurerm_resource_group.admin_rg.name
+  function_app_name   = module.function_admin.name
+  function_app_id     = module.function_admin.id
+  app_service_plan_id = module.function_admin.app_service_plan_id
+  health_check_path   = "/info"
+
+  storage_account_name               = module.function_admin.storage_account.name
+  storage_account_access_key         = module.function_admin.storage_account.primary_access_key
+  internal_storage_connection_string = module.function_admin.storage_account_internal_function.primary_connection_string
+
+  os_type                                  = "linux"
+  linux_fx_version                         = "NODE|14"
+  always_on                                = "true"
+  runtime_version                          = "~4"
+  application_insights_instrumentation_key = data.azurerm_application_insights.application_insights.instrumentation_key
+
+  app_settings = merge(
+    local.function_admin.app_settings_common, {
+      "AzureWebJobs.UpdateVisibleServicesCache.Disabled"             = "1"
+      "AzureWebJobs.UpdateVisibleServicesCacheOrchestrator.Disabled" = "1"
+      "AzureWebJobs.UpdateVisibleServicesCacheActivity.Disabled"     = "1"
```

Comment:
```suggestion
```
like up

Reply:
Fixed in [b253b32](https://github.com/pagopa/io-infra/pull/315/commits/b253b329c7952d66d809818d88378280082e96d6)